tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+import os
+import signal
+import subprocess
+import time
+from contextlib import contextmanager
+from random import randrange
+
+import pytest
+
+
+@contextmanager
+def run_local_server():
+    port = randrange(8010, 8100)
+    subprocess.Popen(
+        "npm run build",
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        shell=True,
+        preexec_fn=os.setsid,
+    )
+    npm_runner = subprocess.Popen(
+        f"npm run watch:eleventy -- --port={port}",
```

Comment:
Question: Why did we switch from `npm start` to `npm run watch:eleventy` under test? Won't watching the filesystem for changes make it slower for no reason?

Reply:
`npm run start` is actually `npm run copyAssets && npm rebuild node-sass && npx npm-run-all build:sass --parallel watch:*`. However, we can't use `--port` when using `watch:*`. 

Even using ` eleventy --serve` will enable a watcher. I think the only way to avoid this is to use Python's HTTP server (or some other HTTP server) on the `_site` directory, and build once. 

I could look into this, but I don't think it's going to be affecting very much.